client/src/pages/Review/CoursePanel.tsx

```diff
@@ -23,7 +39,41 @@ const CoursePanel = ({ course, onChangeAssessment }: CoursePanelProp) => {
 
   return (
     <>
-      <div className="flex flex-col md:flex-row">
+      <div className="z-10">
+        <AppTopBar className="max-w-9xl mx-auto" style={{ paddingLeft: left }} variant="large">
```

Comment:
`AppTopBar` is moved into `CoursePanel` here from `Review`, alone with the `Assessment` list.